### ISSUE TITLE
Feature Request: PIN code to disable selectable sensors for a time slot

### ISSUE THREAD

**Issue by toxynoid** [guardianproject/haven#286]
if i.e. the charwoman comes I want here to be able to disable noise and motion detection for a given time, without being able to move the device.

**Comment by libBletchley** [comment]
Good idea.  But let's take it further.  The device should be controllable in the web interface.  This is the use-case I have in mind:

Owner vacates a house and leaves the country.  They intend to make the house available to guests (friends, air bnb, etc).  The house needs security from intruders during the vacated periods, but when a guest legitimately enters, the guest should have control.

Owners should be able to give guest account access to the onion page.  Guests should have all the same privileges as admins (including ability to see the logs), except admins need to be able to change the guest password.

Guests also need some control against admins who surreptitiously enable/disable sensors.  Perhaps guest visibility on the logs is sufficient (since apparently the logs are not easily erasable). Not sure.

It's worth noting that there have been some Air bnb scandals where owners were caught secretly filming their guests.  So this needs some caution.
